.rubocop.yml

```diff
@@ -1,63 +1,79 @@
+# List of cops can be found here:
+# https://github.com/bbatsov/rubocop/blob/master/config/default.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
+require: rubocop-rspec
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
   DisplayCopNames: true
-  Include:
-    - '**/Rakefile'
-    - '**/config.ru'
+  DisplayStyleGuide: true
   Exclude:
-    - 'db/**/*'
-    - 'script/**/*'
-    - 'spec/internal/**/*'
-    - 'spec/test_app_templates/**/*'
     - 'vendor/**/*'
+    - 'db/**/*'
+    - 'tmp/**/*'
+  ExtraDetails: true
+  TargetRubyVersion: 2.2
 
-Lint/ImplicitStringConcatenation:
-  Exclude:
-    - 'lib/generators/sufia/**/*'
-
-Metrics/LineLength:
-  Max: 400
+# Alot of these metrics are quite restrictive. We may want to disable a few of these.
+# For now let's keep them and increase them by an excessive amount
 
+# A calculated magnitude based on number of assignments,
+# branches, and conditions.
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Max: 45 # default 15
 
+Metrics/ClassLength:
+  Max: 500 # default 100
+
+# A complexity metric that is strongly correlated to the number
+# of test cases needed to validate a method.
+Metrics/CyclomaticComplexity:
+  Max: 17 # default 6
```

Comment:
Are we just making up a number here? Is this worth keeping? I'm not a huge fan of cyclometric complexity tests tbh. It tends to whine about a lot of idiomatic constructs.